akka-http/src/main/scala/akka/http/scaladsl/server/Rejection.scala

```diff
@@ -113,8 +113,7 @@ final case class InvalidOriginRejection(allowedOrigins: immutable.Seq[SHttpOrigi
  */
 final case class UnsupportedRequestContentTypeRejection(supported: immutable.Set[ContentTypeRange])
   extends jserver.UnsupportedRequestContentTypeRejection with Rejection {
-  override def getSupported: java.util.Set[model.ContentTypeRange] =
-    scala.collection.mutable.Set(supported.map(_.asJava).toVector: _*).asJava // TODO optimise
+  override def getSupported: java.util.Set[model.ContentTypeRange] = supported.map(_.asJava).asJava
```

Comment:
Have you tried `supported.asJava`?

Reply:
yes. doesn't work

Reply:
Already simpler this way than before, so :+1: